### PR TITLE
Add note about Single Step Instrumentation version

### PR DESCRIPTION
**Pull request by brett0000FF**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/trace_collection/automatic_instrumentation/ssi-0-13-1.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+kind: documentation
+private: true
+---
+
+The Agent installer script now installs version 0.13.1 of Single Step Instrumentation. Previous versions of Single Step Instrumentation created a temporary file without restricting read permissions. This file contains the environment variables specified when a process is launched. Single Step Instrumentation deletes the file automatically, although in rare circumstances this may not occur. The permissions on this file have been updated. If you find any files in the `/tmp` directory of your Linux hosts whose names start with `dd_process_info_` , we recommend deleting them.
+
+Customers who use Single Step Instrumentation for Linux Hosts should update to the latest version of Single Step Instrumentation by running the Datadog installer script with the environment variable `DD_APM_INSTRUMENTATION_ENABLED=host` :
+
+```
+DD_API_KEY=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX DD_SITE="datadoghq.com" DD_APM_INSTRUMENTATION_ENABLED=host  bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
```
Comment (jonbodner) [inline content/en/tracing/trace_collection/automatic_instrumentation/ssi-0-13-1.md:12]: Should probably make this match the format we use for this line on the main page:

DD_API_KEY=<YOUR_DD_API_KEY> DD_SITE="<YOUR_DD_SITE>" DD_APM_INSTRUMENTATION_ENABLED=host DD_ENV=<AGENT_ENV> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"

**Review by jonbodner — APPROVED** [review]
Make that one minor change and LGTM.

File: content/en/tracing/trace_collection/automatic_instrumentation/ssi-0-13-1.md (hunk 1 above)
Comment (brett0000FF) [reply content/en/tracing/trace_collection/automatic_instrumentation/ssi-0-13-1.md]: Good point. Just committed that change.

File: content/en/tracing/trace_collection/automatic_instrumentation/ssi-0-13-1.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: Single Step Instrumentation v0.13.1
+kind: documentation
+private: true
+---
+
+The Agent installer script now installs version 0.13.1 of Single Step Instrumentation. Previous versions of Single Step Instrumentation created a temporary file without restricting read permissions. This file contains the environment variables specified when a process is launched. Single Step Instrumentation deletes the file automatically, although in rare circumstances this may not occur. The permissions on this file have been updated. If you find any files in the `/tmp` directory of your Linux hosts whose names start with `dd_process_info_` , we recommend deleting them.
+
+Customers who use Single Step Instrumentation for Linux Hosts should update to the latest version of Single Step Instrumentation by running the Datadog installer script with the environment variable `DD_APM_INSTRUMENTATION_ENABLED=host` :
+
+```
```
Comment (hestonhoffman) [inline content/en/tracing/trace_collection/automatic_instrumentation/ssi-0-13-1.md:11]:
~~~suggestion
```sh
~~~
Adding shell will give you a copy button

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**